tools/diagnostics/service_bundle.py

```diff
@@ -149,62 +148,78 @@ def download_log_files(self):
 
     @config.retry
     def create_offers_file(self):
-        scheduler_vip = sdk_hosts.scheduler_vip_host(self.service_name, "api")
-        scheduler = self.scheduler_tasks[0]
+        response = sdk_cmd.service_request("GET", self.service_name, "/v1/debug/offers",
+                                           raise_on_error=False)
+        if not response.ok:
+            log.error(
+                "Could not get scheduler offers\nstatus_code: '%s'\nstderr: '%s'",
+                response.status_code, response.text
+            )
+        else:
+            self.write_file("service_v1_debug_offers.html", response.text)
 
-        rc, stdout, stderr = sdk_cmd.marathon_task_exec(
-            scheduler["id"], "curl -s {}/v1/debug/offers".format(scheduler_vip), print_output=False
-        )
+    @config.retry
+    def create_v2_offers_file(self):
+        response = sdk_cmd.service_request("GET", self.service_name, "/v2/debug/offers",
+                                           raise_on_error=False)
+        if not response.ok:
+            log.error(
+                "Could not get v2 scheduler offers\nstatus_code: '%s'\nstderr: '%s'",
+                response.status_code, response.text
+            )
+        else:
+            self.write_file("service_v2_debug_offers.json", response.text)
 
-        if rc != 0 or stderr:
+    @config.retry
+    def create_plans_file(self):
+        response = sdk_cmd.service_request("GET", self.service_name, "/v1/debug/plans",
+                                           raise_on_error=False)
+        if not response.ok:
             log.error(
-                "Could not get scheduler offers\nstdout: '%s'\nstderr: '%s'", stdout[:100], stderr
+                "Could not get scheduler plans\nstatus_code: '%s'\nstderr: '%s'",
+                response.status_code, response.text
             )
         else:
-            self.write_file("service_v1_debug_offers.html", stdout)
+            self.write_file("service_v1_debug_plans.json", response.text)
+
+    @config.retry
+    def create_taskstatuses_file(self):
+        response = sdk_cmd.service_request("GET", self.service_name, "/v1/debug/taskStatuses",
+                                           raise_on_error=False)
+        if not response.ok:
+            log.error(
+                "Could not get scheduler task-statuses\nstatus_code: '%s'\nstderr: '%s'",
+                response.status_code, response.text
+            )
+        else:
+            self.write_file("service_v1_debug_taskstatuses.json", response.text)
```

Comment:
(nit) how about naming this method and the filename identical to the endpoint as `service_v1_debug_taskStatuses` ?!